<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>590</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="temp_contour1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>30</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temp_contour2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>120</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temp_contour3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>210</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="temp_contour4">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>300</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="lap_stats">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>30</y>
      <width>511</width>
      <height>341</height>
     </rect>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QProgressBar" name="wear_percent">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>380</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string notr="true">%p%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lfw">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>LFW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rfw">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>RFW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lbw">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>LBW</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rbw">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>51</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>RBW</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="seconds_lcd">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>430</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_sec">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+sec</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_sec">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-sec</string>
    </property>
   </widget>
   <widget class="QDial" name="streeting">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-30</number>
    </property>
    <property name="maximum">
     <number>30</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="lap">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>300</y>
      <width>93</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>lap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reset">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>300</y>
      <width>41</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="turn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>300</y>
      <width>93</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string>turn</string>
    </property>
   </widget>
   <widget class="QPushButton" name="plus_speed">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>430</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>+speed</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minus_speed">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>-speed</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="speed_lcd">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>430</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="angle_lcd">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>91</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>-10</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Angle:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Handbrake</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Duration</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>390</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Velocity</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="wear_percent_1">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>410</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string notr="true">%p%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="wear_percent_2">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>440</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string notr="true">%p%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="wear_percent_3">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>470</y>
      <width>511</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="format">
     <string notr="true">%p%</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>615</x>
      <y>10</y>
      <width>21</width>
      <height>531</height>
     </rect>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="laps">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>390</y>
      <width>131</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1360</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
